testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveViews.java

```diff
@@ -516,4 +516,24 @@ public void testCastTimestampAsDecimal()
         onHive().executeQuery("DROP VIEW cast_timestamp_as_decimal_view");
         onHive().executeQuery("DROP TABLE cast_timestamp_as_decimal");
     }
+
+    @Test(groups = HIVE_VIEWS)
+    public void testRunAsInvoker()
+    {
+        onTrino().executeQuery("DROP TABLE IF EXISTS run_as_invoker");
+        onTrino().executeQuery("DROP VIEW IF EXISTS run_as_invoker_view");
+
+        onTrino().executeQuery("CREATE TABLE run_as_invoker (a INTEGER)");
+        onTrino().executeQuery("CREATE VIEW run_as_invoker_view AS SELECT * FROM run_as_invoker");
```

Comment:
Trino views are not "hive views" (views created by hive), which are tested in this class.

I don't think that Trino views are related to the change here, which is about fixing and testing `hive.hive-views.run-as-invoker` toggle's behavior.